src/MicroflowTimer/widget/MicroflowTimer.js

```diff
@@ -113,33 +115,33 @@ define([
 
         _runTimer: function() {
             logger.debug(this.id + "._runTimer", this.interval);
-            if (this.microflow !== "" && this._contextObj) {
+            if (this.callEvent !== "" && this._contextObj) {
```

Comment:
`change to if (this.callEvent === "doNothing" && this._contextObj) {` if you have included the option in the widget `xml`